app/views/content/roadmap.md

```diff
@@ -1,47 +1,42 @@
 <p class="govuk-body-l">Find out about recent and upcoming service changes.</p>
 
-<p class="govuk-hint">Last updated: 9 May 2022</p>
+<p class="govuk-hint">Last updated: 18 October 2022</p>
 
 ***
 
 ### Changes to services for providers
 
 Recent changes include:
 
-* trialling online chat to enable providers to contact Department for Education (DfE) support agents quickly
+* Providing referee details to providers upon application submission
```

Comment:
probably we should be consistent in capitalisation of the first word of these bullet points.